mysql/infoschema.go

```diff
@@ -304,6 +310,120 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+// getForeignKeys return list all the foreign keys constraints.
+// MySQL supports cross-database foreign key constraints. We ignore
+// them because HarbourBridge works database at a time (a specific run
+// of HarbourBridge focuses on a specific database) and so we can't handle
+// them effectively.
+func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (foreignKeys []schema.ForeignKey, err error) {
+	q := `SELECT k.REFERENCED_TABLE_NAME,k.COLUMN_NAME,k.REFERENCED_COLUMN_NAME
+		FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS t 
+		INNER JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS k 
+			ON t.CONSTRAINT_NAME = k.CONSTRAINT_NAME 
+			AND t.CONSTRAINT_SCHEMA = k.CONSTRAINT_SCHEMA 
+			AND t.TABLE_NAME = k.TABLE_NAME 
+			AND k.REFERENCED_TABLE_SCHEMA = k.TABLE_SCHEMA
+		WHERE k.TABLE_SCHEMA = ? 
+			AND k.TABLE_NAME = ? 
+			AND t.CONSTRAINT_TYPE = "FOREIGN KEY" 
+		ORDER BY
+			k.REFERENCED_TABLE_NAME,
+			k.ordinal_position;`
```

Comment:
Let's be consistent in our capitalization of queries.
i.e. replace k.ordinal_position with k.ORDINAL_POSITION.

Reply:
updated the query, also updated postgres query similarly